shared/reducers/wallets.js

```diff
@@ -59,11 +59,6 @@ export default function(state: Types.State = initialState, action: WalletsGen.Ac
         if (action.payload.accountID) {
           stateMutable.update('currencyMap', c => c.set(action.payload.accountID, action.payload.currency))
         }
-        if (action.payload.setBuildingCurrency) {
```

Comment:
Left these action changes in because I think it's best to only have this set in one place

Reply:
Thing is, lastSentXLM isn't applied anywhere else, so it's broken now

Reply:
whoops! Making new pr...